setup/pos_payment_entries_globalization/odoo/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Shouldn't this setup files go away and be added automatically after merge?

Reply:
It is as no influence, the generated one will the same. We add it in the PR to be able to use pip even if not merged.